kr/kr.go

```diff
@@ -84,6 +84,28 @@ func pairCommand(c *cli.Context) (err error) {
 	return pairOver(kr.DaemonSocketOrFatal(), c.Bool("force"), nameOpt, os.Stdout, os.Stderr)
 }
 
+func pairCommandForce() (err error) {
```

Comment:
can we prevent analytics when being invoked by kr transfer? if this is only used in kr transfer then analytics can just be deleted

Reply:
only in transfer, will remove analytics line 👍